app/src/main/java/nu/yona/app/api/receiver/YonaReceiver.java

```diff
@@ -76,7 +76,7 @@ private void handleDeviceDozeMode(Context context)
 		PowerManager powerManager = (PowerManager) context.getSystemService(POWER_SERVICE);
 		if (powerManager.isDeviceIdleMode())
```

Comment:
The proposed changes feature is rather incomplete at the moment, as it doesn't allow doing a multiline change. You still need to remove this line and the related curly brace, and reduce the indentation.

Reply:
yes. noticed and corrected in the update below.